book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -155,7 +216,8 @@ In summary:
 - **Sled reads** may be done synchronously (in `call`) or asynchronously (in
   the `Future`), depending on the context;
 
-- **Sled writes** must be done synchronously (in `call`).
+- **Sled writes** must be done synchronously (in `call`), which is guaranteed
+  by the state's external API (a `Buffer`ed `tower::Service`).
```

Comment:
I don't think that the new text is accurate -- the state's external API being a buffered Service does *not* guarantees that Sled writes are done synchronously. We have to implement all Sled writes synchronously, nothing will guarantee that for us.